content/changelog-stable-old/index.html.haml

```diff
@@ -11,15 +11,23 @@ actions:
 ---
 
 .ratings
-  - # Load older changelogs (all except the most recent 25, ordered correctly)
+  - # Load older changelogs (all except the most recent 25, ordered correctly with full entries list)
   - archived_changelogs = site.changelogs[:lts].reverse.drop(25)
   - archived_changelogs.each do |release|
     %div
       = partial('../_partials/release-header.html.haml', :release => release, :url => 'changelog-stable-old')
       - if release.changes
+        .app-releases__notable-changes{:style => "margin-top: 15px"}
+          = "Changes since #{release.lts_baseline}"
         %div.app-releases__list__items
           = partial('../_partials/changelog-changes.html.haml', :changes => release.changes)
-      - else
+      - if release.lts_changes
+        .app-releases__notable-changes
+          - if release.lts_predecessor
```

Comment:
The first release of an LTS line (like 2.375.1, 2.479.1, 2.492.1) needs the "Changes since <LTS baseline>".  Subsequent releases of an LTS line (like 2.375.2 and 2.479.3) should not have "Changes since".

Something seems incorrect about the results of this conditional.  

I see the following on the [2.375.3 page](https://deploy-preview-7891--jenkins-io-site-pr.netlify.app/changelog-stable-old/#v2.375.3):

![Screenshot 2025-02-17 091921](https://github.com/user-attachments/assets/2d1f121b-9df6-47a0-ae2a-62c8fbe3c3a4)